### PR TITLE
UI: Implement print tables and messages

### LINKED ISSUE #28: UI: Implement print tables and messages
- [x] Add horizontal lines above and below all responses
- [x] Refactor `println` to `printResponse`
- [x] Implement `printStudentList`, `printAllocatingMessage`, `generateReport`
- [x] Create enum for general constant messages
- [x] Create feedback messages for specific commands (add, delete)
- [x] Fix ASCII table formatting

### PR DESCRIPTION
**Pull request by isaacsaw25**
-  Add horizontal lines above and below all responses
-  Refactor println to printResponse
-  Implement printStudentList, printAllocatingMessage, generateReport
-  Create enum for general constant messages
-  Create feedback messages for specific commands (add, delete)
-  Fix ASCII table formatting

Fix #28

### CODE REVIEW

File: src/main/java/command/HelpCommand.java [hunk 1]
```diff
@@ -13,6 +14,6 @@ public HelpCommand(StudentList studentList, UI ui) {
 
     @Override
     public void run() {
-        this.ui.printHelp();
+        this.ui.printResponse(Messages.HELP);
```
Comment (ehz0ah) [inline src/main/java/command/HelpCommand.java:17]: Good use of Enum.

File: src/main/java/student/Student.java [hunk 2]
```diff
@@ -12,7 +12,7 @@ public class Student {
     public Student(String id, float gpa, ArrayList<Integer> uniPreferences){
         this.id = id;
         this.gpa = gpa;
-        this.uniPreferences = new ArrayList<>();
+        this.uniPreferences = uniPreferences;
```
Comment (ehz0ah) [inline src/main/java/student/Student.java:15]: Good catch! 😂

**Review by ehz0ah — APPROVED** [review]
Good work :)

**Comment by ehz0ah** [comment]
Remember to add the appropriate labels, etc.

**Review by Holy-An — APPROVED** [review]
Good job!

**Review by Holy-An — APPROVED** [review]

**Pull request merged**